Packs/CommonScripts/Scripts/UnzipFile/UnzipFile.yml

```diff
@@ -36,15 +36,15 @@ outputs:
   description: A list of file names that were extracted from the ZIP file.
   type: Unknown
 script: '-'
+subtype: python3
 system: false
 tags:
 - Utility
 - file
 timeout: '0'
 type: python
-dockerimage: demisto/7zip
+dockerimage: devdemisto/unzip:1.0.0.10195
 runonce: false
 tests:
 - ZipFile-Test
 - UnzipFile-Test
-subtype: python2
```

Comment:
i'm not sure you can change the subtype of the automation. if it is a must, we might need to do a v2.

Reply:
I don't think it's a must. Just figured we would bring it to py3. I can gen the docker image as a python2 image without any issues. Question is if we want to do `toVersion` --> `fromVersion` and just quietly replace as a v2. @yuvalbenshalom what are your thoughts?

Reply:
because this is a simple one, it is ok just to test it with regular zip, 7zip and password, and just add `dockerimage45`. no need for toversion/fromversion.